include/radix/core/math/Vector2ui.hpp

```diff
@@ -39,10 +39,6 @@ struct Vector2ui {
     return x != v.x || y != v.y;
   }
 
-  constexpr Vector2ui operator-() const {
-    return Vector2ui(-x, -y);
-  }
-
```

Comment:
Why does this need to be removed?

Reply:
The result is undefined and Visual Studio can't compile it. Basically it is:
```
unsigned int foo = 4;
foo = -foo;
```
There is no way to represent a negative number in an unsigned int

Reply:
@kungfooman Thanks for clarifying. I've just looked at the whole vector class and to me it does not make sense to change the sign to negative for an usigned int, too. And I remember I was probably the person to introduce this bug since vector2ui is mostly a copy and paste of vector2i.